tests-with-coverage/README.md

```diff
@@ -0,0 +1,658 @@
+# Measuring the coverage of your tests
+
+Learn how to measure the test coverage of your application. This guide covers:
+
+* Measuring the coverage of your Unit Tests
+* Measuring the coverage of your Integration Tests
+* Separating the execution of your Unit Tests and Integration Tests
+* Consolidating the coverage for all your tests
+
+Please note that code coverage is not supported in native mode
+
+## Prerequisites
+
+To complete this guide, you need:
+
+* less than 15 minutes
+* an IDE
+* JDK 1.8+ installed with JAVA_HOME configured appropriately
+* Apache Maven 3.5.3+
+* Having completed the guide [Geeting started testing](https://quarkus.io/guides/getting-started-testing)
+
+## Architecture
+
+The application built in this guide is just a JAX-RS endpoint (hello world) that relies on dependency injection to use a service.
+Service will be tested thanks to JUnit 5 and Endpoint will be tested via a ```@QuarkusTest``` annotation.
+
+## Solution
+
+We recommend that you follow the instructions in the next sections and create the application step by step. However, you can go right to the completed example.
+Clone the Git repository: git clone {quickstarts-clone-url}, or download an {quickstarts-archive-url}[archive].
+The solution is located in the tests-with-coverage directory.
+
+## Starting from a simple project and two tests
+
+Let's start with an application that looks like the content of the guide *getting-started-testing* but with a few modifications for the sake of the example.
```

Comment:
Frankly, I would start from a Maven command generating a new application. I don't think it's a good idea to base this one on the other one.

Reply:
I was a bit puzzled to find how to start. Let's start from scratch like you suggest.

Reply:
Modification done. Tell me if that's what you expected.